docs/source/_static/css/copybtn.css

```diff
@@ -0,0 +1,23 @@
+.highlight {
+    position: relative;
+}
+
+.copybtn {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 40px;
+    padding: 0 10px;
```

Comment:
can we add bottom padding ?
![image](https://user-images.githubusercontent.com/5157099/131245892-30b031ab-bfd3-4bf0-868c-39cc594624e1.png)